safe/gui/tools/wizard/test/test_wizard_util.py

```diff
@@ -0,0 +1,14 @@
+# coding=utf-8
+
+"""Test for Wizard Util.
+"""
+
+import unittest
+
+
+class TestWizardUtil(unittest.TestCase):
+    pass
+
```

Comment:
So, there is already this `wizard_util.py`, but no test file for it. I added some methods in `wizard_util.py`, create `test_wizard_util.py` for my methods.